gp-templates/profile-public.php

```diff
@@ -24,7 +24,7 @@
 				}
 			?></dd>
 			<dt><?php _e( 'Member Since', 'glotpress' ); ?></dt>
-			<dd><?php echo date( 'M j, Y', strtotime( $user->user_registered ) ); ?></dd>
+			<dd><?php /* translators: public profile date format */ echo date_i18n( __( 'M j, Y', 'glotpress' ), strtotime( $user->user_registered ) ); ?></dd>
```

Comment:
@toolstack Thanks for the info, I made formatting changes and added the WPCS comment. I hope I did it correct this time (the WPCS comment is new for me in this context). And yes, I agree with the readability :)

As for the comment vs. usage of _x() --- as you all said, comment would be enough for now, only if we have more instances of this date appearance we should consider _x(). I currently did not run in other appearances of dates on the frontend so far... :)